src/js/components/PageHeader/index.d.ts

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import {
+  A11yTitleType,
+  AlignSelfType,
+  GridAreaType,
+  MarginType,
+} from '../../utils';
+
+export interface PageHeaderProps {
+  a11yTitle?: A11yTitleType;
+  alignSelf?: AlignSelfType;
+  gridArea?: GridAreaType;
+  margin?: MarginType;
```

Comment:
Can this be removed? I don't see it being consumed anywhere.

Reply:
These are the prop types for what we pass onto all components as "genericProps"